agent/engine/docker_image_manager_test.go

```diff
@@ -1055,7 +1073,9 @@ func TestConcurrentRemoveUnusedImages(t *testing.T) {
 	for i := 0; i < numRoutines; i++ {
 		go func() {
 			<-ok
-			imageManager.removeUnusedImages()
+			ctx, cancel := context.WithCancel(context.TODO())
```

Comment:
can you move the context creation and `defer cancel()` methods outside of the `for` loop?

Reply:
OK, here only one context should created and used.